core/service/src/builder.rs

```diff
@@ -1095,6 +1098,9 @@ ServiceBuilder<
 				endpoints,
 				wasm_external_transport: config.telemetry_external_transport.take(),
 			});
+			let startup_time = SystemTime::UNIX_EPOCH.elapsed()
+				.map(|dur| dur.as_millis())
```

Comment:
Isn't seconds enough?

Reply:
This would pass machine start time, right? We want node (process) run time.

Reply:
Telemetry should show node running time regardless of when we connected to telemetry. E.g. if the node was started an hour ago, and it only connected to telemetry 15 minutes ago, it should still show "1 hour". For this to work we need to pass our real startup time I believe.

Reply:
This code is called only once, when we build the service.
`startup_time` is then cloned (well, copy'ed) when we connect to the telemetry.

Reply:
I misread it

Reply:
> Isn't seconds enough?

@maciejhirsz told me milliseconds were better.